lib/localvore_sdk.ex

```diff
@@ -54,6 +54,8 @@ defmodule LocalvoreSdk do
     ] ++ headers
   end
 
+  defp process_response_body(""), do: ""
```

Comment:
Agreed.  I think it's best to handle all of the reasonable error cases.  The `nil` return seems like a good idea as well 👍 

```elixir
defp process_response_body(nil), do: nil
defp process_response_body(""), do: nil
```